api/go.sum

```diff
@@ -54,8 +54,8 @@ github.com/pmezard/go-difflib v1.0.0 h1:4DBwDE0NGyQoBHbLQYPwSUPoCMWR5BEzIk/f1lZb
 github.com/pmezard/go-difflib v1.0.0/go.mod h1:iKH77koFhYxTK1pcRnkKkqfTogsbg7gZNVY4sRDYZ/4=
 github.com/prashantv/gostub v1.1.0 h1:BTyx3RfQjRHnUWaGF9oQos79AlQ5k8WNktv7VGvVH4g=
 github.com/prashantv/gostub v1.1.0/go.mod h1:A5zLQHz7ieHGG7is6LLXLz7I8+3LZzsrV0P1IAHhP5U=
-github.com/rkvst/go-rkvstcommon v0.4.0 h1:kt0F7eP73ybfSUkVbWXJtFm3XbIO0FKvuSjBjs0Q0+E=
-github.com/rkvst/go-rkvstcommon v0.4.0/go.mod h1:M0Mi6v/2Jgbc5se2zWKbShZC9bTVI7eFQE/xTi5Q9J8=
+github.com/datatrails/go-datatrails-common v0.4.0 h1:kt0F7eP73ybfSUkVbWXJtFm3XbIO0FKvuSjBjs0Q0+E=
+github.com/datatrails/go-datatrails- v0.4.0/go.mod h1:M0Mi6v/2Jgbc5se2zWKbShZC9bTVI7eFQE/xTi5Q9J8=
```

Comment:
was this changed manually ? Normally one changes go.mod and then executes 'go mod tidy', 'go mod verify' but I presume you dont have go installed locally

Reply:
manually updated but had issues running go mod as discussed